graphql-dxm-provider/pom.xml

```diff
@@ -77,6 +77,12 @@
         <jahia-module-signature>MCwCFBcjFkCyiSxf9LM8Zqox6+L5CrJ3AhRHAP0Tnka4B+4h2SH06EVSCh2J9w==</jahia-module-signature>
         <mockito.version>2.21.0</mockito.version>
         <powermock.version>2.0.7</powermock.version>
+
+
+        <graphql-java.version>13.0</graphql-java.version>
```

Comment:
Do we need this here? or should be coming from jahia-parent?

Reply:
no need, can be removed

Reply:
Removed in latest commit